packages/@expo/config-types/src/ExpoConfig.ts

```diff
@@ -1,8 +1,15 @@
 /* tslint:disable */
+
+export type RuntimeVersionPolicyType =
```

Comment:
These changes need to be made in www xdl-schemas, I just did them here to ease review. Will revert and change upstream and import before landing this.